examples/text_to_image/train_text_to_image_mlflow.py

```diff
@@ -917,7 +905,7 @@ def collate_fn(examples):
                     noisy_latents = noise_scheduler.add_noise(latents, noise, timesteps)
 
                 # Get the text embedding for conditioning
-                encoder_hidden_states = text_encoder(batch["input_ids"])[0]
+                encoder_hidden_states = text_encoder(batch["input_ids"])[0].to(dtype=weight_dtype)
```

Comment:
`text_encoder` is already casted to `weight_dtype`, why do you need to cast one more time here?

Reply:
You're right. It's redundant